src/main/java/lotto/LottoNumberGenerator.java

```diff
@@ -0,0 +1,19 @@
+package lotto;
+
+import java.util.Collections;
+import java.util.List;
+import java.util.stream.Collectors;
+import java.util.stream.IntStream;
+
+public class LottoNumberGenerator {
+    private static final List<Integer> NUMBER_POOL = IntStream.range(1, 46)
```

Comment:
값을 하드코딩하기보다는 상수로 관리하는 것이 유지보수하기에 좋다고 생각합니다.
상수(static final)를 만들고 이름을 부여하면 이 변수의 역할이 무엇인지 의도를 드러내기에 좋다고 생각합니다.
자바 컨벤션 9.6 Constants를 확인해보세요.

https://myeonguni.tistory.com/1596

Reply:
```suggestion
    private static final List<Integer> NUMBER_POOL = IntStream.rangeClosed(1, 45)
```
위의 범위로 표현하는 것이 더 직관적일 거 같네요